app/(dashboard)/dashboard/setup/page.tsx

```diff
@@ -0,0 +1,31 @@
+import { AutomationsList } from "./components/AutomationsList";
+import { SetupDialog } from "./components/SetupDialog";
+import { getUserRepositoryConfigs } from "@/lib/db/queries";
+
+export default async function SetupPage() {
+  const configs = await getUserRepositoryConfigs();
+
+  return (
+    <div className="space-y-6 h-full flex flex-col">
+      <div className="p-6">
+        <div className="flex items-center justify-between">
+          <div>
+            <h1 className="text-2xl font-semibold tracking-tight">
+              QA Automations
```

Comment:
This is more about managing / viewing your projects. Check Vercel.com UI for inspiration.